### PR TITLE
Subworkflow2

### PR DESCRIPTION
**Pull request by sysbiocoder**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR> -stub`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/analyse_MT.nf [hunk 1]
```diff
@@ -15,15 +14,19 @@ workflow ANALYSE_MT {
         dict          // channel: [ genome.dict ]
```
Comment (Lucpen) [inline subworkflows/local/analyse_MT.nf:14]:
```suggestion
        sequence_dict          // channel: [ genome.dict ]
```

Comment (Lucpen) [reply subworkflows/local/analyse_MT.nf:14]: If dict has been changed to sequence_dict on align_and_call_MT.nf, then it should be done here too

File: subworkflows/local/analyse_MT.nf [hunk 2]
```diff
@@ -33,14 +36,41 @@ workflow ANALYSE_MT {
             dict,
```
Comment (Lucpen) [inline subworkflows/local/analyse_MT.nf:36]:
```suggestion
            sequence_dict,
```

File: workflows/raredisease.nf [hunk 3]
```diff
@@ -323,7 +333,7 @@ workflow RAREDISEASE {
     ch_multiqc_files = ch_multiqc_files.mix(Channel.from(ch_multiqc_config))
     ch_multiqc_files = ch_multiqc_files.mix(ch_multiqc_custom_config.collect().ifEmpty([]))
     ch_multiqc_files = ch_multiqc_files.mix(ch_workflow_summary.collectFile(name: 'workflow_summary_mqc.yaml'))
-    ch_multiqc_files = ch_multiqc_files.mix(CUSTOM_DUMPSOFTWAREVERSIONS.out.mqc_yml.collect())
+    //ch_multiqc_files = ch_multiqc_files.mix(CUSTOM_DUMPSOFTWAREVERSIONS.out.mqc_yml.collect())
```
Comment (Lucpen) [inline workflows/raredisease.nf:336]:
```suggestion
    ch_multiqc_files = ch_multiqc_files.mix(CUSTOM_DUMPSOFTWAREVERSIONS.out.mqc_yml.collect())
```

File: workflows/raredisease.nf [hunk 4]
```diff
@@ -89,6 +92,7 @@ include { ANNOTATE_CSQ as ANN_CSQ_SV   } from '../subworkflows/local/annotate_co
 
 include { FASTQC                      } from '../modules/nf-core/modules/fastqc/main'
 include { MULTIQC                     } from '../modules/nf-core/modules/multiqc/main'
+
 include { CUSTOM_DUMPSOFTWAREVERSIONS } from '../modules/nf-core/modules/custom/dumpsoftwareversions/main'
```
Comment (Lucpen) [inline workflows/raredisease.nf:96]:
```suggestion
include { MULTIQC                     } from '../modules/nf-core/modules/multiqc/main'
include { CUSTOM_DUMPSOFTWAREVERSIONS } from '../modules/nf-core/modules/custom/dumpsoftwareversions/main'
```

File: workflows/raredisease.nf [hunk 5]
```diff
@@ -68,6 +70,7 @@ include { FILTER_VEP as FILTER_VEP_SV  } from '../modules/local/filter_vep'
 
 include { CHECK_INPUT                  } from '../subworkflows/local/check_input'
 include { PREPARE_REFERENCES           } from '../subworkflows/local/prepare_references'
+
 include { ANNOTATE_SNVS                } from '../subworkflows/local/annotate_snvs'
```
Comment (Lucpen) [inline workflows/raredisease.nf:74]:
```suggestion
include { PREPARE_REFERENCES           } from '../subworkflows/local/prepare_references'
include { ANNOTATE_SNVS                } from '../subworkflows/local/annotate_snvs'
```

**Review by Lucpen — COMMENTED** [review]
Good job! The LiftOver needs to be added to the analyse_MT.nf

**Comment by sysbiocoder** [comment]
Reopened with liftover

**Pull request closed without merging**